acrobat/blocks/verb-widget/verb-widget.js

```diff
@@ -1,10 +1,35 @@
 import LIMITS from './limits.js';
-import { setLibs } from '../../scripts/utils.js';
+import { setLibs, getEnv, isOldBrowser } from '../../scripts/utils.js';
 import verbAnalytics from '../../scripts/alloy/verb-widget.js';
 
 const miloLibs = setLibs('/libs');
 const { createTag } = await import(`${miloLibs}/utils/utils.js`);
 
+const fallBack = 'https://www.adobe.com/go/acrobat-overview';
+const EOLBrowserPage = 'https://acrobat.adobe.com/home/index-browser-eol.html';
+
+const verbRedirMap = {
+  createpdf: 'createpdf',
+  'crop-pages': 'crop',
+  'delete-pages': 'deletepages',
+  'extract-pages': 'extract',
+  'combine-pdf': 'combine',
+  'protect-pdf': 'protect',
+  'add-comment': 'addcomment',
+  'pdf-to-image': 'pdftoimage',
+  'reorder-pages': 'reorderpages',
+  sendforsignature: 'sendforsignature',
+  'rotate-pages': 'rotatepages',
+  fillsign: 'fillsign',
+  'split-pdf': 'split',
+  'insert-pdf': 'insert',
+  'compress-pdf': 'compress',
+  'png-to-pdf': 'jpgtopdf',
+  'number-pages': 'number',
+  'ocr-pdf': 'ocr',
+  'chat-pdf': 'chat',
```

Comment:
How about study-with-acrobat?

Reply:
Thanks @TsayAdobe , i"ve added it: https://github.com/adobecom/dc/pull/800/commits/ed5d2b4fc656e781491597d74ba819a81c565dd4